include/pq_flash_index.h

```diff
@@ -123,25 +125,25 @@ template <typename T, typename LabelT = uint32_t> class PQFlashIndex
     float max_base_norm = 0.0f;
 
     // data info
-    _u64 num_points = 0;
-    _u64 num_frozen_points = 0;
-    _u64 frozen_location = 0;
-    _u64 data_dim = 0;
-    _u64 disk_data_dim = 0; // will be different from data_dim only if we use
-                            // PQ for disk data (very large dimensionality)
-    _u64 aligned_dim = 0;
-    _u64 disk_bytes_per_point = 0;
+    size_t num_points = 0;
```

Comment:
these should also be uint64_t? i am thinking only file sizes, memory, offsets etc shd be size_t?